app/presenters/base.presenter.js

```diff
@@ -218,6 +218,43 @@ function formatPounds(valueInPence) {
   return valueInPounds.toFixed(2)
 }
 
+/**
+ * Format the purposes for display on screen
+ *
+ * This function assumes that when provided the purposes are in the format:
+ *
+ * ```javascript
+ * [{
+ *  alias: 'manual description',
+ *  primary: { code: 'A', description: 'Agriculture' },
+ *  tertiary: { code: '400', description: 'Spray Irrigation - Direct' },
+ *  secondary: { code: 'AGR', description: 'General Agriculture' }
+ * }]
+ * ```
+ *
+ * It is possible that a return log has zero or more purposes associated it. If this is the case we need the description
+ * of each purpose from the array of purposes to be concatenated into a single comma separated string ready to present.
+ *
+ * @param {Array} purposes - the purposes taken from the return logs metadata
+ *
+ * @returns {string} the purpose descriptions as a string, separated by commas if more than one description exists
+ */
+function formatPurposes(purposes) {
+  if (!purposes || purposes.length === 0) {
+    return ''
+  }
+
+  const purposeDescriptionArray = purposes.map((purpose) => {
+    if (purpose.alias) {
+      return `${purpose.tertiary.description} (${purpose.alias})`
```

Comment:
Just wondering where the requirement came from to concatenate the "description" and "alias" if an "alias" exists? Previously we appeared to just show one or the other. Concatenating is going to make the string considerably longer, don't know if this impacts the formatting on any of the pages this is displayed 🤷🏼‍♂️

Reply:
This came up in conversation with chris when we were triaging the bug last week. If the alias is there it should be displayed as it is more important as it is something provided by the abstractor. I've checked it on all of the pages I'm familiar with and added screen shots to the story but yeah I meant to ask you how to get to that bill page.

Reply:
Fair enough. I'd still probably just display either the alias or the description rather than concatenate them together as we did previously `return purpose.alias ? purpose.alias : purpose.tertiary.description`. Unless we have been asked specifically to concatenate the two together. Just think it may make the string a bit messy.